Dockerfile

```diff
@@ -46,10 +46,12 @@ FROM ros-desktop AS sr-ros2-bundles
 # default value of arg ELASTICA when not provided in the --build-arg
 ARG ELASTICA=false
 ARG PYTORCH=false
+ARG SOFA=false
 
 # install some general system dependencies
-RUN apt update && apt install -y --no-install-recommends \
-    iputils-ping libeigen3-dev python3-pip python3-tk &&\
+RUN apt update && apt install -y --no-install-recommends software-properties-common \
+    iputils-ping libeigen3-dev python3-pip python3-tk wget unzip zip &&\
+    add-apt-repository -y ppa:deadsnakes/ppa && apt install -y libpython3.7 && add-apt-repository --remove -y ppa:deadsnakes/ppa &&\
```

Comment:
@marcomerton Can you explain me this line? Why do we need to install python3.7?

Reply:
It is requird by Sofa binaries, since they have been compiled with that version of python. Trying with another version of python gives errors when loading the sofa python plugin

Reply:
@marcomerton Ok, this is a bit unfortunate. I hope it will not interfere with Python 3.8 binary used in ROS. Can you please add a code comment explaining why you added the installation of `libpython3.7`?

Reply:
I though the same. That's why I wanted to test things properly before really merging things.
I'll add the comment anyway, so in case some problems arises, there's track the problem could be that

Reply:
@marcomerton You could (potentially) also put the code of the libpython3.7 installation into an if clause for `SOFA=true`

Reply:
Yeah, good idea. Done